content/general/concepts/algorithm/BigO.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'Big-O Notation'
+Description: 'An algorithm is a formal process used to solve a problem. To compare two algorithms, it is necessary to develop tools to evaluate algorithmic efficiency. This comparison is very important since while many algorithms might function in the same way, they may also consume a differing amount of resources. It is often best practice to limit the footprint of the algorithms that are developed and to do that, Big-O notation is to be used. '
```

Comment:
```suggestion
Description: 'Big-O notation is a form of measuring the complexity of an algorithm. '
```

Reply:
The Description of an entry should A) Immediately reference the term in question and B) Be relatively short and pithy